content/blog/eng/different-types-of-software-testing.md

```diff
@@ -0,0 +1,130 @@
+finds out by running some tests: if we put it this way it is clear that focusing on **enhancing code quality**, analyzing the 
+features before/alongside development and having the right amount of tests are good things to do.
+
+Software testing is however an engineering practice that offers support to different areas of software development. 
+There are therefore different methodologies, tools and needs to adopt the most suitable testing activities for your IT world
+
+
+## Types
+
+So how do we plan test design and how can we decide which types of tests suits our application?  
+
+There are *many types* of **software tests** and different classifications or topologies are defined of which 
+I report an example in the following image:
+
+![test Types](https://ehikioya.azureedge.net/wp-content/uploads/hm_bbpui/86828/fh2aspgg9au0icjnuf8d9lvhwccrv977.png)  
+
+
+### Acceptance tests
+
+Acceptance tests are based on business requirements, this type of tests checks whether the end-to-end process works as intended by the business or user requirements.
+We often see this type of tests as the last step of the software delivery cycle: whether it is run automatically in a CI or performed manually by a Developer or QA.
+The software is usually accepted when the feature works as expected by the requirements.
+
+### Functional tests
+
+This type of tests checks that a specific feature functionality is working as intended by the business or user requirement. Differently from the previously seen **Acceptance Tests* this type does not rely on a fully functioning application as it only tests a specific feature of it.
+An example of functional test may be checking that a login component is working as expected.
+
+### Smoke tests
+
+Smoke tests are usually run before executing other types of tests. This type is very basic as it checks the application is working from high level perspective.
+These tests can check, for example, that a website's main page is loaded correctly while different types of tests will check the functionalities right after.
+Smoke tests should be quick and easy to both automate and maintain.
+
+### Integration tests
+
+These tests usually verify that different sections and services of an application work together as expected. This type of test is usually quite easy to automate and maintain but can be very expensive due to the requirement of having different services up and running.
+
+### Unit tests
+
+Unit are source-level tests which are executed on an individual portion of the application code: these tests are typically automated and run by software developers.
+Unit tests verify that the specific unit/function respects the intended design and works as expected.
+This type of tests is usually seen automated and executed in continuous integration servers.
+
+### Performance tests
+
+Performance tests verify if the application performance is compromised during high loads. This type of tests split in mupltiple sub-types:
+
+```
```

Comment:
We should use the quoting format `> ` for this piece from Wiki

Reply:
@Jean85 done